tools/lint/lint.py

```diff
@@ -747,8 +747,19 @@ def changed_files(wpt_root):
 
 def lint_paths(kwargs, wpt_root):
     if kwargs.get("paths"):
-        r = os.path.realpath(wpt_root)
-        paths = [os.path.relpath(os.path.realpath(x), r) for x in kwargs["paths"]]
+        #make new list in case there are multiple added directories
+        AllPaths=[]
+        for path in kwargs.get("paths"):
+            # create extension for called directory and merge it with the existing path
+            extension= path+'/'
+            newPath=(str(os.path.realpath(wpt_root))+'/'+ extension)
+            # get all files from new path
+            paths = list(all_filesystem_paths(newPath))
+            # add the extension to the strings for all files from the new path,
+            # so they may be called from the current directory
+            paths=list(map(lambda path : extension + path, paths))
```

Comment:
A list comprehension is more idomatic here, so this would look like `[extension + path for path in paths]`. That said, it isn't clear what you're doing here. In the clause below the results of `all_filesystem_paths` are used directly, so it seems like it should be fine to do that here too.

Reply:
I tried that initially, but it was insufficient because it assumes we are already in the directory passed to ```all_filesystem_paths``` and will not work unless we are already operating from that directory